### ISSUE TITLE
Issue with menu roles.

### ISSUE THREAD

**Issue by airliajsmith** [coreui/coreui-free-laravel-admin-template#48]
I have two roles for my account: Employee and Admin.

I can't see the menu elements that have only the role Employee. If I remove all roles from my account except Employee, I can see the menu. Why is that?

**Comment by rakieta2015** [comment]
If you want a menu item to be visible for both roles, enter both roles into the menu item in MenuTableSeeder.php in this way:
$this->insertLink('Admin,Employee', 'Cards', '/base/cards');
(In answer to the question: This is because the role of Admin is higher in the hierarchy than role Employee.)

**Comment by airliajsmith** [comment]
Sorry, I thought I had this figured out. 

In the UsersSeeder, I entered the role 'order-entry into menuroles column. Then, I assign the role of 'order-entry' via the $user->assignRole('order-entry'). 

Then, in the MenusTableSeeder, I insert this:

      $this->beginDropdown('employee', 'Employee Links', 'cil-folder-open');

        $this->insertLink('order-entry', 'Sales Orders', '/salesorders');
        $this->insertLink('order-entry', 'Phone List', '/phonelist');
        $this->endDropdown();

However, when I login, I can't see the menu link I just inserted. 

I am thinking about having our company purchase the pro version, but I need to get this figured out first.

Thanks.
